lib/lists/helpers.js

```diff
@@ -0,0 +1,58 @@
+import _ from 'lodash';
+import logger from '../utils/log';
+
+const log = logger(__filename);
+
+/**
+ * Get the format of the items in a list
+ * @param {array} item
+ * @returns {string}
+ */
+
+function getFormat(items) {
+  const tester = _.head(items);
+
+  if (_.isString(tester)) {
+    return 'STRING';
+  } else if (_.has(tester,'text')) {
+    return 'OBJECT';
+  } else {
+    log.error('List cannot be modified because it is not a simple list.', { action: 'modifyList' });
+  }
+}
+
+/**
+ * @param {array} items
+ * @param {string} testItem
+ * @returns {integer}
+ */
+export function getItemIndex(items, testItem) {
+  // force tester to be a string
+  const tester = _.isString(testItem) ? testItem : testItem.text,
```

Comment:
Should this call `getFormat` here, rather than below? (since that function handles the case where it's neither a string nor object)